packages/cli/src/lib/implementation/only-plugins.utils.ts

```diff
@@ -14,12 +14,16 @@ export function filterPluginsBySlug(
 // skip the whole category if it has at least one skipped plugin ref
 // see https://github.com/code-pushup/cli/pull/246#discussion_r1392274281
 export function filterCategoryByPluginSlug(
-  categories: CoreConfig['categories'],
+  categories: Array<CategoryConfig>,
```

Comment:
For consistency:

```suggestion
  categories: CategoryConfig[],
```

Pre-emptive answer: Yes, there is a lint rule for this ([array-type](https://typescript-eslint.io/rules/array-type/)) :grin: I've created an issue for adding more of these stylistic rules to our ESLint config: https://github.com/code-pushup/eslint-config/issues/4